inspire/modules/deposit/workflows/literature.py

```diff
@@ -527,6 +527,14 @@ def restructure_ffts(fft):
             submission_number=deposition.id,
         )
 
+        # ==============
+        # Extra comments
+        # ==============
+        if 'extra_comments' in metadata and metadata['extra_comments']:
+            metadata['hidden_note'] = {'value': metadata['extra_comments'],
+                                       'submitter': userid}
+            delete_keys.append('extra_comments')
```

Comment:
if you delete the key ``extra_comments`` then they will be not available to be shown in the Holding Pen detailed page:

https://github.com/inspirehep/inspire-next/blob/master/inspire/modules/workflows/templates/workflows/details.html#L45